editoast/src/main.rs

```diff
@@ -717,6 +747,92 @@ mod tests {
         tmp_file
     }
 
+    #[actix_test]
+    async fn clone_infra_returns_new_cloned_infra() {
```

Comment:
We have some fixtures that provide an Infra. That way you won't have to cleanup manually. Checkout `fixtures.rs`